### PR TITLE
fix(chat): unify the modal windows paddings (Issue #1191)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Fixed modal paddings

Issues:

- Issue #1191 

**UI changes**

Need to correct:

Attachments
Снимок экрана 2024-05-16 в 10 50 37

Confirm clearing
Снимок экрана 2024-05-16 в 10 51 03

Share
Снимок экрана 2024-05-16 в 10 51 19

Some items failed to import due to duplicate names ( also add external paddings)
Снимок экрана 2024-05-16 в 10 51 43

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by ai-dial-actions** [comment]
Hey there and thank you for opening this pull request! 👋🏼

We require pull request titles to follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your proposed title needs to be adjusted.

Details:

```
No release type found in pull request title "Fix/1191 modal paddings". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/

Available types:
 - fix
 - hotfix
 - feat
 - feature
 - chore
```

File: apps/chat/src/components/Chat/ShareModal.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default function ShareModal() {
   return (
     <Modal
       portalId="theme-main"
-      containerClassName="inline-block w-full max-w-[424px] p-6"
+      containerClassName="inline-block w-full max-w-[424px] px-3 py-4 md:p-6"
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ShareModal.tsx]: Can we move `px-3 py-4 md:p-6` in base component `Modal` instead?

File: apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx [hunk 2]
```diff
@@ -299,11 +299,10 @@ export const ReplaceConfirmationModal = ({ isOpen }: Props) => {
         'flex size-full flex-col sm:w-[525px]',
         featuresToReplace.length < 3 && 'sm:h-fit',
       )}
-      overlayClassName="px-0 py-0"
       dismissProps={{ outsidePressEvent: 'mousedown' }}
     >
       <div className="flex h-full flex-col justify-between gap-2 sm:gap-4">
-        <div className="flex h-[90%] flex-col gap-4 p-6 pb-0">
+        <div className="flex h-[90%] flex-col gap-4 px-3 py-4 pb-0 md:p-6">
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: why do we have `padding-bottom:0 (pb-0)` here?

File: apps/chat/src/components/Chat/ShareModal.tsx (hunk 1 above)
Comment (Derikyan) [reply apps/chat/src/components/Chat/ShareModal.tsx]: I removed individual styles from other places and added this code to the modal in a new commit

File: apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx (hunk 2 above)
Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: this was previously written, I fixed it in the new commit

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/Modal.tsx [hunk 3]
```diff
@@ -102,7 +102,7 @@ function ModalView({
           >
             <Tag
               className={classNames(
-                'relative max-h-full rounded bg-layer-3 text-left',
+                'relative max-h-full rounded bg-layer-3 px-3 py-4 text-left',
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Common/Modal.tsx]: need to be careful. For instance manage dialog attachments has huge paddings right now

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1383.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**